### PR TITLE
Add tests for FileContent func

### PR DESCRIPTION
**Pull request by ialidzhikov**
**What this PR does / why we need it**:
In the past we had a small regression with this FileContent func - ref gardener/gardener#2008. The PR adds test to prevent similar case.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: pkg/chartrenderer/default_test.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	})
+
+	Describe("#Render", func() {
+		It("should return err when chartPath is missing", func() {
+			_, err := renderer.Render(filepath.Join("testdata", "missing"), "missing", "default", map[string]string{})
+			Expect(err).To(HaveOccurred())
+		})
+
+		It("should return rendered chart", func() {
+			chart, err := renderer.Render(alpineChartPath, "missing", "default", map[string]string{})
```
Comment (timuthy) [inline pkg/chartrenderer/default_test.go]:
```suggestion
			chart, err := renderer.Render(alpineChartPath, "alpine", "default", map[string]string{})
```

File: pkg/chartrenderer/default_test.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+			files := chart.Files()
+			Expect(files).To(HaveLen(1))
+			Expect(files).To(HaveKeyWithValue("alpine/templates/alpine-pod.yaml", alpinePod))
+		})
+	})
+
+	Describe("#FileContent", func() {
+		It("should return empty string when template file is missing", func() {
+			chart, err := renderer.Render(alpineChartPath, "alpine", "default", map[string]string{})
```
Comment (timuthy) [inline pkg/chartrenderer/default_test.go:86]:
```suggestion
			chart, err := renderer.Render(alpineChartPath, "missing", "default", map[string]string{})
```

**Review by timuthy — COMMENTED** [review]
Nice! Looks good. Only one proposal (nit) to be more consistent with the chart names.

File: pkg/chartrenderer/default_test.go (hunk 2 above)
Comment (ialidzhikov) [reply pkg/chartrenderer/default_test.go:86]: The missing in this test is is not the chartPath but the filename passed to FileContent func.

**Comment by plkokanov** [comment]
Looks good!

File: pkg/chartrenderer/default_test.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+				Major: "1",
+				Minor: "16",
+			},
+		}
+
+		capabilities, err := chartrenderer.DiscoverCapabilities(fakeDiscovery)
+		Expect(err).ToNot(HaveOccurred())
+
+		renderer = chartrenderer.New(engine.New(), capabilities)
+		Expect(err).ToNot(HaveOccurred())
```
Comment (timebertt) [inline pkg/chartrenderer/default_test.go]: No error returned in the previous call.
```suggestion
```

**Review by timebertt — CHANGES_REQUESTED** [review]
Thanks, only one nit :)

File: pkg/chartrenderer/default_test.go (hunk 3 above)
Comment (ialidzhikov) [reply pkg/chartrenderer/default_test.go]: Nice catch, thank you!

**Review by rfranzke — APPROVED** [review]

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**